src/sqlfluff/core/parser/helpers.py

```diff
@@ -28,9 +28,7 @@ def check_still_complete(
     current_str = join_segments_raw(matched_segments + unmatched_segments)
     if initial_str != current_str:  # pragma: no cover
         raise RuntimeError(
-            "Dropped elements in sequence matching! {!r} != {!r}".format(
-                initial_str, current_str
-            )
+            f"Dropped elements in sequence matching! {initial_str} != {current_str}"
```

Comment:
Don't see a problem with this change per se, but any reason for the change?

Reply:
I just was looking on that method and spotted that _previous_ formatting style. Generally it's nothing wrong with using `.format()` but f-strings are more readable and whenever I can I change that.